fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/perception/SimplePerceptionModule.java

```diff
@@ -127,9 +133,53 @@ public boolean isInRange(SpatialObject other) {
                 other.getProjectedPosition().toVector3d(tmpVector1).subtract(origin);
                 // we use tmpVector2 as origin from the viewpoint of this object
                 tmpVector2.set(0, 0, 0);
-                return VectorUtils.isLeftOfLine(tmpVector1, tmpVector2, rightBoundVector) // vehicle is left of right edge
-                        && !VectorUtils.isLeftOfLine(tmpVector1, tmpVector2, leftBoundVector) // vehicle is right of left edge
-                        && tmpVector1.magnitude() <= configuration.getViewingRange(); // other vehicle is in range
+
+                if (tmpVector1.magnitude() > configuration.getViewingRange()) { // other vehicle is in range
```

Comment:
sorry for being picky, but the comment is the negative of the condition 🤐

Reply:
Good Point! Fixed this